projects/packages/search/src/customberg/class-customberg.php

```diff
@@ -44,6 +44,8 @@ public static function instance() {
 	 */
 	public function init_hooks() {
 		add_action( 'admin_menu', array( $this, 'add_wp_admin_page' ), 999 );
+		add_action( 'pre_option_jetpack_search_show_powered_by', array( $this, 'get_show_powered_by' ) );
```

Comment:
should this be a filter? Also is this necessary since we've already forced the display [here](https://github.com/Automattic/jetpack/blob/7e10c8f7387cd65a5de7ec37df36acf37d94fbb7/projects/packages/search/src/class-helper.php#L850).

Reply:
Yes, it should be a filter. Fixed!

I wish we didn't need this, but it's necessary for overriding values returned by [useSearchOptions](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/search/src/customberg/hooks/use-search-options.js#L29), which reads the option values directly for previewing on Customberg.